xrdp/lang.c

```diff
@@ -233,7 +233,11 @@ get_keymaps(int keylayout, struct xrdp_keymap *keymap)
     /* check if there is a keymap file */
     g_snprintf(filename, 255, "%s/km-%4.4x.ini", XRDP_CFG_PATH, keylayout);
 
-    /* if the file does not exist, try again with 'en-us' as fallback */
+    /* if the file does not exist, try the local default, and finally 'en-us' as fallback */
+    if (!g_file_exist(filename))
+    {
+        g_snprintf(filename, 255, "%s/km-default.ini", XRDP_CFG_PATH);
```

Comment:
This creates an undocumented feature. Comments in the code are not documentation.